src/util/address.rs

```diff
@@ -870,6 +888,29 @@ impl fmt::Debug for Address {
     }
 }
 
+/// decode address to Bech32 u8 byte array
+fn bech32_decode(address: &Address) -> Result<Vec<u8>, Error> {
```

Comment:
I gave this MR another read through since I effectively wrote it ages ago. This function isn't actually needed. It is effectively just `as_payload_bytes()` so it has been dropped